src/platform/Darwin/system/SystemLayerImplDispatch.mm

```diff
@@ -32,6 +32,19 @@
 
 #define SYSTEM_LAYER_IMPL_DISPATCH_DEBUG 0
 
+// Note: CONFIG_BUILD_FOR_HOST_UNIT_TEST
+//
+// Certain unit tests are executed without a main dispatch queue, relying instead on a mock clock
```

Comment:
Need documentation in `src/platform/Darwin/system/SystemLayerImplDispatchSockets.mm` as well, right?